tools/raisinbread_refresh.php

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+/**
+ * This script is intended for developers working with test data. It DROPs all
+ * LORIS core tables as well as "Raisinbread" test instrument tables.
+ * For this reason, it should obviously not be run on any server with live data.
+ *
+ * The `sandbox` flag in config.xml is checked for this very reason. The script
+ * will abort if it is not set to 1.
+ *
+ * After dropping the tables, the script will source the Raisinbread test data
+ * using the commands found in raisinbread/README.md.
+ *
+ * The script also restores the url, base, and host config settings to their
+ * pre-drop values. This prevents an issue where a developer will need to
+ * manually change these values in a MySQL shell when they are not hosting a
+ * LORIS on localhost.
+ *
+ * Finally, the script runs tools/resetpassword.php so that the default admin
+ * password with the Raisinbread data set is not used.
```

Comment:
I don't understand this part.. does this mean every time someone runs this script the password on raisinbread changes?

Reply:
thtas the thing. yes if the users table is then submitted back to the repo... might be problematic

Reply:
When you source Raisinbread, your admin user will have its password set to the public password contained in the Raisinbread dataset. This comment explains that the script will run `php resetpassword.php admin` and force you to change your password for your dev environment to a new, private password.

Yes there are potential complications when submitting changes to Raisinbread, but only if the `users` table is updated which is the one containing the password. In this case, the author of the PR will need to change the RB file manually back to the public RB admin password. This is similar to the process whent he Config table is updated; the author needs to change e.g. `host` back to localhost in their PR.

Reply:
nothing prevents us from handling these fields in the dump script to make sure they are reset before the db is dumped !! that could be a nice addition beyond the scope of this PR

Reply:
Yeah I agree with you. I think it's a modification to the exporter script, not this one.